edb/lib/schema.edgeql

```diff
@@ -47,6 +47,11 @@ CREATE ABSTRACT TYPE schema::Object {
 CREATE ABSTRACT TYPE schema::Type EXTENDING schema::Object;
 CREATE TYPE schema::PseudoType EXTENDING schema::Type;
 
+ALTER TYPE schema::Type {
```

Comment:
I don't quite like using schema::Type for this. Can we add schema::ExpressionAlias?

Reply:
Well, those objects *are* types (technically abstract type variants), so this introspection reflection is actually true. Separating them would require a whole bunch of hacks, because you would need to replicate the whole `Type` hierarchy only in the `Alias` form, i.e. `ObjectAlias`, `TupleAlias` etc. I actually tried going in that direction and it went ugly real fast.

Reply:
ok